src/components/SearchInput.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useMemo } from 'react'
+import PropTypes from 'prop-types'
+import debounce from 'lodash/debounce'
+import InputGroup from 'cozy-ui/transpiled/react/InputGroup'
+import Input from 'cozy-ui/transpiled/react/Input'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import makeStyles from 'cozy-ui/transpiled/react/helpers/makeStyles'
+import MagnifierIcon from 'cozy-ui/transpiled/react/Icons/Magnifier'
+
+const useStyles = makeStyles(theme => ({
+  input: {
+    borderRadius: '1.25rem',
+    maxWidth: 'none',
+    height: '2.5rem',
+    flex: '1',
+    boxShadow: theme.shadows[1],
+    border: '0',
+    '&:hover, &:focus, &:active': {
+      border: '0',
+      boxShadow: theme.shadows[6]
+    },
+    '& input': {
+      maxWidth: 'none',
+      padding: '.625rem 1rem',
+      borderRadius: '1.25rem'
+    }
+  }
+}))
+
+const SearchInput = ({ placeholder, setValue }) => {
+  const styles = useStyles()
+
+  const delayedSetValue = useMemo(
```

Comment:
I think using `useCallback` would be better in this case. (let's discuss if you want precision)

Reply:
c'est pareil, sauf que ça ne convient pas à notre linter il me semble (déjà eu sur un autre repo). Je pense surtout qu'on peut supprimer le memo ici en fait 🤔 

ps : tout ça c'est du nit à mon avis...